CONTRIBUTING.md

```diff
@@ -0,0 +1,152 @@
+# Contributing to opentelemetry-ebpf-profiler
+
+The Profiling special interest group (SIG) meets regularly. See the
+OpenTelemetry
+[community](https://github.com/open-telemetry/community)
+repo for information on this and other SIGs.
+
+See the [public meeting
+notes](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit#heading=h.4rdgawyis2hd)
+for a summary description of past meetings.
+
+See the [calendar
+group](https://groups.google.com/a/opentelemetry.io/g/calendar-profiling) to
+get invited to meetings.
+
+See the [#otel-profiles](https://cloud-native.slack.com/archives/C03J794L0BV)
+slack channel for discussions and questions.
```

Comment:
Wondering whether we should move this down, into a dedicated new section, e.g. "Community". Starting the document with this may give the impression that one has to take part in these meetings in order to contribute?